runtime/Python3/src/antlr4/IntervalSet.py

```diff
@@ -8,6 +8,29 @@
 import unittest
 from antlr4.Token import Token
 
+class Interval(object):
+
+    def __init__(self, start, stop):
+        self.start = start
+        self.stop = stop
+        self.range = list(range(start, stop))
+
+    def __contains__(self, item):
+        return item in self.range
+
+    def __len__(self):
+        return self.stop - self.start
+
+    def __iter__(self):
+        return iter(self.range)
+
+    def __getitem__(self, idx):
+        if idx == 0:
+            return self.start
+        elif idx == 1:
+            return self.stop
+        raise IndexError('Interval index out or range [{}]'.format(idx))
+
```

Comment:
I have noticed that in python2-runtime, there's this ``Interval`` class inside ``IntervalSet`` module. However, the ``Interval`` class is missing in python3-runtime.
It is sort of strange.